### DISCUSSION TITLE
Deleted from nuget gallery

### DISCUSSION THREAD

**Discussion by mrcoymeeks** [linmasaki/Kendo.DynamicLinqCore#22] [Q&A] [answered]
Hi,

I've noticed that the package has been deleted from the nuget gallery.  Is there a reason for that?  Mistake or is this package going offline permanently?

Thanks for the work!

**Comment by linmasaki** [comment, marked as answer]
Since the prefix "Kendo" is registered by Progress Software Corporation, this package can no longer be named Kendo.DynamicLinqCore. Currently, I'm trying to rename it and republish it. Thanks for your reply!

**Reply by stevengwebsolutions** [reply]
any chance of getting the 3.1.1 source code that has the async ToDataSourceResult posted?  The NuGet package we had downloaded was 3.1.1 and had that functionality.  Thanks.

**Reply by linmasaki** [reply]
Hello @stevengwebsolutions, it is good to get your feedback. I will publish and rename it in a few days, so please be patient. I hope you can understand. :pensive:

**Comment by mrcoymeeks** [comment]
I figured it was something along those lines.  Thanks again!

**Comment by linmasaki** [comment]
Hello @mrcoymeeks & @stevengwebsolutions, I republished it and its name now is [KendoNET.DynamicLinq](https://www.nuget.org/packages/KendoNET.DynamicLinq)
